doc/man1/openssl-ca.pod.in

```diff
@@ -142,9 +142,10 @@ F<.pem> appended.
 
 The CA certificate file.
 
-=item B<-certform> B<DER>|B<PEM>
+=item B<-certform> B<DER>|B<PEM>|B<P12>
 
 The format of the data in certificate input files.
+This option has no effect and is retained for backward compatibility only.
```

Comment:
Did this change deserve a word in History sections below ?

Reply:
Good point - done. Also for x509, s_client, s_server, and for the `xcertform` option.